.github/maintainers_guide.md

```diff
@@ -105,6 +107,24 @@ Run a specific integration test:
 $ ./scripts/run_integration_tests.sh integration_tests/web/test_async_web_client.py
 ```
 
+#### Develop Locally
+
+If you want to test the package locally you can.
+
+1. Build the package locally
+   - Run
+     ```bash
+     scripts/build_pypi_package.sh
+     ```
+   - This will create a `.whl` file in the `./dist` folder
+2. Use the built package
+   - Example `/dist/slack_sdk-1.2.3-py2.py3-none-any.whl` was created
+   - From anywhere on your machine you can install this package to a project with
+     ```bash
+     pip install <project path>/dist/slack_sdk-1.2.3-py2.py3-none-any.whl
+     ```
+   - It is also possible to include `<project path>/dist/slack_sdk-1.2.3-py2.py3-none-any.whl` in a [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files) file
```

Comment:
Small NIT I lean today, pip understands the dependency better with `slack_sdk @` prefix
```suggestion
   - It is also possible to include `slack_sdk @ file:///<project path>/dist/slack_sdk-1.2.3-py2.py3-none-any.whl` in a [requirements.txt](https://pip.pypa.io/en/stable/user_guide/#requirements-files) file
```

Reply:
TIL! That helps me understand the path better too 📚 ✨